src/main/java/org/folio/service/export/LocalFileSystemExportService.java

```diff
@@ -55,6 +55,13 @@ public void export(List<String> jsonRecords, FileDefinition fileDefinition) {
 
   @Override
   public void postExport(FileDefinition fileDefinition, String tenantId) {
-    exportStorageService.storeFile(fileDefinition, tenantId);
+    if (isValidFileDefinition(fileDefinition)) {
+      exportStorageService.storeFile(fileDefinition, tenantId);
+    }
+  }
+
```

Comment:
Add javadoc

Reply:
The javadoc is present in the method contract, should I add it to the implementation as well?